### ISSUE TITLE
Typescript compiler errror `Cannot find name 'F'.` when compiling flagsmith in Angular project

### ISSUE THREAD

**Issue by baffioso** [Flagsmith/flagsmith-js-client#224]
I'm using flagsmith in a Angular project but are getting following typescript compile error:
```
✘ [ERROR] TS2304: Cannot find name 'F'. [plugin angular-compiler]

    node_modules/flagsmith/types.d.ts:72:46:
      72 │ ...hange = (previousFlags: IFlags<F> | null, params: IRetrieveInfo...
```

Workaround is to set `"skipLibCheck": true` in `tsconfig` but would like to avoid this.

**Comment by kyle-ssg** [comment]
This is fixed in 4.0.1 of flagsmith, flagsmith-es and react-native-flagsmith. Please re-open this if you are still having any issues. Thanks again 🚀 .

**Comment by baffioso** [comment]
Works like a charm. Thanks for the quick fix ⏩
